db/migrate/20200527123642_ideal_transactions_to_payment.rb

```diff
@@ -0,0 +1,13 @@
+class IdealTransactionsToPayment < ActiveRecord::Migration[6.0]
+  def change
+    rename_table :ideal_transactions, :payments
+    rename_column :payments, :redirect_uri, :ideal_redirect_uri
+
+    remove_column :payments, :transaction_type, :string
+
+    add_column :payments, :transaction_type, :integer, default: 0
+    add_column :payments, :payment_type, :integer, default: 0
+
+    change_column :checkout_transactions, :payment_method, :integer
```

Comment:
this doesn't work, just change the limit of payment_method and remove payment_type